shanoir-ng-import/src/main/java/org/shanoir/ng/importer/dicom/query/QueryPACSService.java

```diff
@@ -522,14 +522,8 @@ private void querySeries(Association association, Study study, DicomParam modali
 	private void processDICOMSerie(Attributes serieAttr, Association association, Study study, DicomParam modality, List<Serie> series) {
 		Serie serie = new Serie(serieAttr);
 		if (!DicomSerieAndInstanceAnalyzer.checkSerieIsIgnored(serieAttr)) {
-			if (serie.getNumberOfSeriesRelatedInstances() > 0) {
```

Comment:
Hi @LaurentPV, thank you for this PR.
I wonder about this removal here. Normally, every PACS returns numberOfSeriesRelatedInstances, normally.
Is there a specific reason to remove it? Maybe to show it to the user in the tree? as empty. With kind regards, Michael

Reply:
answered in other PR